pkg/cache/cache.go

```diff
@@ -0,0 +1,23 @@
+package cache
+
+import (
+	"time"
+)
+
+type Cache struct {
+	lastRefreshTime time.Time
+	bridges         map[string]bool
+}
+
+func (c *Cache) Refresh(freshBridges map[string]bool) {
+	c.bridges = freshBridges
+	c.lastRefreshTime = time.Now()
+}
+
+func (c *Cache) LastRefreshTime() time.Time {
+	return c.lastRefreshTime
+}
+
+func (c Cache) Bridges() map[string]bool {
+	return c.bridges
```

Comment:
We have to return a new map here to prevent cache being modified without using "Refresh"

You can check the possible problem here https://play.golang.org/p/-h9fqv2gdII

Reply:
nice catch!  and thanks for the playground explanation.
Done.